models/issue_watch.go

```diff
@@ -91,6 +91,9 @@ func GetIssueWatchers(issueID int64) ([]*IssueWatch, error) {
 func getIssueWatchers(e Engine, issueID int64) (watches []*IssueWatch, err error) {
 	err = e.
 		Where("issue_id = ?", issueID).
```

Comment:
Same here, should have `watch.issue_id`

Reply:
As mentioned on the other comment, I can add it for the sake of clarity, but it wouldn't be necessary.